modules/fedimint-meta-tests/src/bin/meta-sanity-test.rs

```diff
@@ -0,0 +1,185 @@
+use std::future::Future;
+
+use anyhow::{bail, Result};
+use devimint::federation::Client;
+use devimint::util::poll_simple;
+use devimint::{cmd, util};
+use fedimint_core::PeerId;
+use semver::Version;
+use serde_json::json;
+use tracing::{info, warn};
+
+#[tokio::main]
+async fn main() -> anyhow::Result<()> {
+    devimint::run_devfed_test(|dev_fed| async move {
```

Comment:
I would move the closure to a outer function, just to reduce a level of indentation